tooling/noir_js_backend_barretenberg/src/types.ts

```diff
@@ -5,4 +5,5 @@
 export type BackendOptions = {
   /** @description Number of threads */
   threads: number;
+  memory?: { maximum: number };
```

Comment:
Should we add a `/** @description ... */` to supplement the [BackendOptions doc page](https://noir-lang.org/docs/dev/reference/noirjs/backend_barretenberg/type-aliases/backendoptions/#type-declaration)?

Describing e.g. this being the maximum memory limit, unit of the number (MB? GB?), etc.